src/runtime/pkg/oci/utils_test.go

```diff
@@ -1014,3 +1014,68 @@ func TestParseAnnotationBoolConfiguration(t *testing.T) {
 		}
 	}
 }
+
+func TestParseLinuxDevice(t *testing.T) {
+	assert := assert.New(t)
+
+	type parseResult struct {
+		hostPath      string
+		containerPath string
+		permissions   string
+	}
+
+	testCases := []struct {
+		annotaionValue string
+		err            bool
+		result         parseResult
+	}{
+		{
+			annotaionValue: "",
+			err:            true,
+			result:         parseResult{},
+		},
+		{
```

Comment:
Can you add some tests with things like an:

- `/this/path/is/invalid`
- `../../../../`
- `.`
- `..`
- `/`
- unreadable path? (I'm thinking rootless / non-priv)